config/src/main/java/com/redhat/lightblue/config/LightblueFactory.java

```diff
@@ -345,15 +346,15 @@ public Factory getFactory()
         return factory;
     }
 
-    public Mediator getMediator()
+    public Mediator getMediator(RequestMetrics metrics)
```

Comment:
It doesn't really make sense for this method to accept a RequestMetrics. If you call it again, it's just going to use the first metrics that every called this method. Also, everything else that the Mediator uses is initialized elsewhere.

I would probably accept a RequestMetrics within the LightblueFactory constructor instead.

See also: https://github.com/lightblue-platform/lightblue-rest/pull/300#discussion_r134321943